src/main/java/uk/gov/justice/hmpps/prison/api/resource/BookingResource.java

```diff
@@ -444,7 +444,7 @@ public List<Assessment> getAssessments(@PathVariable("bookingId") @Parameter(des
         return inmateService.getAssessments(bookingId);
     }
 
-    @Operation(summary = "Offender case notes.", description = "Offender case notes.")
+    @Operation(summary = "Offender case notes.", description = "Offender case notes.", hidden = true)
```

Comment:
These endpoint shouldn't be used by clients (only case note service)

Reply:
Maybe add  a new role that only the case notes service has to restrict access?

Reply:
That would make sense - obs not in this PR